zest/releaser/pypi.py

```diff
@@ -407,6 +407,31 @@ def version_levels(self):
             return default
         return result
 
+    def tag_format(self):
+        """Return the string formating the tag name in the release.
+
+        Override the default ``%(version)`` in setup.cfg using
+        a ``tag-format`` option::
+
+            [zest.releaser]
+            tag-format = v%(version)s
+
+        Returns default of ``%(version)s`` when nothing has been configured.
+        """
+        default = '%(version)s'
+        if self.config is None:
+            return default
+        try:
```

Comment:
Better unwind the two try/excepts. So a "try/except NoSectionError" (with `return default`) and then on the same level the "try/except KeyError".